lib/Runtime/ByteCode/ByteCodeGenerator.cpp

```diff
@@ -1391,13 +1398,16 @@ FuncInfo * ByteCodeGenerator::StartBindFunction(const char16 *name, uint nameLen
     if (pnode->sxFnc.CallsEval())
     {
         funcInfo->SetCallsEval(true);
+        
         bodyScope->SetIsDynamic(true);
         bodyScope->SetIsObject();
         bodyScope->SetCapturesAll(true);
         bodyScope->SetMustInstantiate(true);
+
+        paramScope->SetIsDynamic(true);
```

Comment:
Is it true that the use of a default expression makes the parameter list obey strict mode rules? Does that extend to the behavior of eval? If so, then the paramScope is never dynamic, because eval can't leak declarations.